packages/mobile/src/account/Persona.tsx

```diff
@@ -24,7 +24,7 @@ export interface Props {
 }
 
 const Persona = ({ kycStatus }: Props) => {
-  const { t } = useTranslation(Namespaces.accountScreen10)
+  const { t } = useTranslation(Namespaces.translation)
   const [personaAccountCreated, setPersonaAccountCreated] = useState(!!kycStatus)
 
   const accountAddress = useSelector(accountAddressSelector)
```

Comment:
accountAddress defaults to the EOA address, which won't work. IHL requires this to be the mtwAddress. It would be better to use the mtwAddressSelector here.